core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -64,4 +67,102 @@ public void numMaxConnections() {
+                .virtualHost(new VirtualHostBuilder("def.example5.com", "*.example5.com")
+                                     .accessLogger("com.ex5").build())
+                .build();
+        assertThat(sb.config().defaultVirtualHost()).isNotNull();
+        assertThat(sb.config().defaultVirtualHost().accessLogger().getName()).isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example.com").accessLogger().getName())
+                .isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example2.com").accessLogger().getName())
+                .isEqualTo("com.ex2");
+
+        assertThat(sb.config().findVirtualHost("*.example3.com").accessLogger().getName())
+                .isEqualTo("com.ex3");
+
+        assertThat(sb.config().findVirtualHost("*.example4.com").accessLogger().getName())
+                .isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example5.com").accessLogger().getName())
+                .isEqualTo("com.ex5");
+    }
+
+    /**
+     * Makes sure that {@link VirtualHost}s can have a proper {@link Logger} used for writing access
+     * when a user specify the default access logger via {@link ServerBuilder#accessLogger(String)}.
+     */
+    @Test
+    public void setAccessLoggerTest2() {
+        final Server sb = new ServerBuilder()
+                .service("/", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .accessLogger("test.default")
+                .withVirtualHost("*.example.com")
+                .and()
+                .build();
+        assertThat(sb.config().defaultVirtualHost().accessLogger().getName())
+                .isEqualTo("test.default");
+        assertThat(sb.config().findVirtualHost("*.example.com").accessLogger().getName())
+                .isEqualTo("test.default");
+    }
+
+    /**
+     * Makes sure that {@link VirtualHost}s can have a proper {@link Logger} used for writing access
+     * when a user doesn't specify it.
+     */
+    @Test
+    public void defaultAccessLoggerTest() {
+        final Server sb = new ServerBuilder()
+                .service("/", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .withVirtualHost("*.example.com")
+                .and()
+                .withVirtualHost("*.example2.com")
+                .and()
+                .build();
+        assertThat(sb.config().findVirtualHost("*.example.com").accessLogger().getName())
+                .isEqualTo("com.linecorp.armeria.logging.access.com.example");
+        assertThat(sb.config().findVirtualHost("*.example2.com").accessLogger().getName())
+                .isEqualTo("com.linecorp.armeria.logging.access.com.example2");
+    }
+
+    /**
+     * Makes sure that {@link ServerBuilder#build()} throws {@link IllegalStateException}
+     * when the access logger of {@link VirtualHost} set by a user is {@code null}.å
```

Comment:
`the access logger of {@link VirtualHost}` -> `the access logger of a {@link VirtualHost}`?
And please remove `å`. :-)